src/features/orcidlink/images.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Constructs urls for images used in the orcidlink ui
+ *
+ * Note that the usual technique, importing the image, which creates a url back
+ * into the web app, does not work with the current state of Europa dependencies.
+ * Therefore, we use hardcoded image paths, combined with the PUBLIC_URL.
+ * When Europa deps are updated, or it is switched to vite, the image imports
+ * should be re-enabled.
+ */
```

Comment:
This relates to the error:

```
ERROR in ./src/features/orcidlink/assets/ORCID-iD_icon-vector.svg
Module build failed (from ./node_modules/file-loader/dist/cjs.js):
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:68:19)
    at Object.createHash (node:crypto:138:10)
    at getHashDigest (/workspace/node_modules/loader-utils/lib/getHashDigest.js:46:34)
    at /workspace/node_modules/loader-utils/lib/interpolateName.js:113:11
    at String.replace (<anonymous>)
    at interpolateName (/workspace/node_modules/loader-utils/lib/interpolateName.js:110:8)
    at Object.loader (/workspace/node_modules/file-loader/dist/index.js:29:48)
```

which occurs when trying to import the image directly. This is related to the libraries used to load the image and security restrictions in node > 16. Some have said it was fixed by CRA updates (and the transitive dependency upon webpack), others say not. There are other workarounds, but they are insecure.

Reply:
Agreed that we should move to vite. Especially with this issue you note about asset loading, that could eventually be a true pain. I opened a PR a while back with an initial attempt. This may be a good thing to bring up as future work to prioritize in the UI meeting.